src/pls/models/node.py

```diff
@@ -17,6 +17,7 @@
 from pls.models.composables.stat import StatComp
 from pls.models.composables.type import TypeComp
 from pls.models.format_rules import FormatRules
+from pls.models.node_spec import NodeSpec
```

Comment:
This can be moved into the `if TYPE_CHECKING` block below as it's only used for type annotation.